data/birth-dates/15.txt

```diff
@@ -1 +1 @@
-Sergey Alexeyevich Lebedev: 2 November ????
+John Forbes Nash Jr.: June 13, 1928
```

Comment:
That does not look correct. The name should not be modified. Please fix only the birth year.